whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumSpec.groovy

```diff
@@ -6191,4 +6191,67 @@ class InetnumSpec extends BaseQueryUpdateSpec {
         then:
         created =~ /Create FAILED: \[inetnum] 192.168.0.0 - 192.168.0.255/
     }
+
+    def "Child creation breaks the hierarchy then error"() {
+        given:
+        databaseHelper.addObject("""\
```

Comment:
Use an existing assignment from the transient array using getTransient() if possible, then submit via syncupdates so we're sure the rules have been followed

Reply:
Done